RELEASES.md

```diff
@@ -80,6 +80,8 @@ Libraries
   `swap_bytes`, `from_be`, `from_le`, `to_be`, `to_le` are now `const` for all
   numeric types.][57234]
 - [`Ipv4Addr::new` is now a `const` function][57234]
+- [Libtest no longer creates a new thread for each test when
+  `--test-threads=1`.  It also runs the tests in deterministic order][56243]
```

Comment:
Could you move this to the compatibility notes section?

I don't think it's really a "feature"